edx_event_bus_kafka/internal/consumer.py

```diff
@@ -28,12 +28,12 @@
 
 # .. toggle_name: EVENT_BUS_KAFKA_CONSUMERS_ENABLED
 # .. toggle_implementation: SettingToggle
-# .. toggle_default: False
-# .. toggle_description: Enables the ability to listen and process events from the Kafka event bus
-# .. toggle_use_cases: opt_in
+# .. toggle_default: True
+# .. toggle_description: If set to False, consumer will exit immediately. This can be used as an emergency kill-switch
+#   to disable a consumer—as long as the management command is killed and restarted when settings change.
+# .. toggle_use_cases: opt_out
 # .. toggle_creation_date: 2022-01-31
-# .. toggle_tickets: https://openedx.atlassian.net/browse/ARCHBOM-1992
-KAFKA_CONSUMERS_ENABLED = SettingToggle('EVENT_BUS_KAFKA_CONSUMERS_ENABLED', default=False)
```

Comment:
Author note: This is currently checked in the `Command`, but it's possible that `consume_indefinitely` could be called some other way. Should the check be moved from the former to the latter?

Reply:
Hm. It's a good point, but if we don't have all the proper settings setup then the command will fail even before it gets to calling `consume_indefinitely` so we do need some kind of switch at the top